cfgov/v1/forms.py

```diff
@@ -160,6 +174,37 @@ def clean(self):
             self.data = data
         return self.cleaned_data
 
+    def set_interpreted_date_values(self, cleaned_data):
+        from_date = cleaned_data.get('from_date')
+        to_date = cleaned_data.get('to_date')
+        # if from_ or to_ is filled in, fill them both with sensible values
+        # if neither is filled in, leave them both blank
+        if from_date or to_date:
+            if from_date:
+                self.data['from_date'] = date.strftime(
+                    cleaned_data['from_date'], '%m/%d/%Y')
+            else:
+                # If there's a 'to_date' and no 'from_date',
+                #  use date of earliest possible filter result as 'from_date'
+                earliest_results = self.first_page_date()
+                cleaned_data['from_date'] = earliest_results
+                self.data['from_date'] = date.strftime(
+                    earliest_results, '%m/%d/%Y')
+
+            if to_date:
+                transformed_to_date = end_of_time_period(
+                    self.data['to_date'], cleaned_data['to_date'])
+                cleaned_data['to_date'] = transformed_to_date
+                self.data['to_date'] = date.strftime(
+                    transformed_to_date, '%m/%d/%Y')
+            else:
+                # If there's a 'from_date' but no 'to_date', use today's date
+                today = date.today()
```

Comment:
Are there any timezone considerations to be had here?

Reply:
The website servers are set to EST/EDT, so this date will always be in that timezone.